src/test/java/uk/gov/pay/directdebit/payers/resources/PayerResourceIT.java

```diff
@@ -142,8 +142,6 @@ public void shouldCreateAPayer_forGoCardless() throws JsonProcessingException {
         insertTransactionFixtureWith(GOCARDLESS);
         String fakeCustomerId = "CU000358S3A2FP";
         String fakeBankAccountId = "BA0002WR3Z193A";
-        stubCreateCustomer(testPaymentRequest.getExternalId(), payerFixture, fakeCustomerId);
-        stubCreateCustomerBankAccount(testPaymentRequest.getExternalId(), payerFixture, fakeCustomerId, fakeBankAccountId);
         String requestPath = "/v1/api/accounts/{accountId}/payment-requests/{paymentRequestExternalId}/payers"
```

Comment:
This still tests the old `POST` endpoint — do we want to test the `PUT` endpoint instead or as well?

Reply:
Yes indeed, should test the new one as the POST will be removed. Fixing it now